src/ray/core_worker/task_manager.cc

```diff
@@ -1002,14 +1002,17 @@ bool TaskManager::RetryTaskIfPossible(const TaskID &task_id,
         RAY_CHECK(num_retries_left == 0);
       }
     }
+    // Keep `num_retries_left` and `num_oom_retries_left` up to date
```

Comment:
`it->second.num_retries_left` and `it->second.num_oom_retries_left` are updated in L989 and L998.